PROTO_tests/tests/numeric_test.py

```diff
@@ -121,8 +128,9 @@ def test_str_cast_errors(self, num_type):
         assert valid.to_list() == validans.to_list()
         assert np.allclose(ans, res.to_ndarray(), equal_nan=True)
 
-    def test_histogram(self):
-        pda = ak.randint(10, 30, 40)
+    @pytest.mark.parametrize("num_type", INT_FLOAT)
+    def test_histogram(self, num_type):
```

Comment:
kinda crazy to me that histogram isn't implemented for `uint`. I feel like we should have that. I just added issue #2521 for this, we can update this test to be `NO_BOOL` as part of that